subsys/bluetooth/controller/Kconfig.df

```diff
@@ -0,0 +1,64 @@
+# Zephyr Bluetooth Controller configuration options
+
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_CTLR_DF_SUPPORT
+	bool
+
+config BT_CTLR_DF_CTE_TX_SUPPORT
+	bool
+
+config BT_CTLR_DF_ANT_SWITCHING_2US_SUPPORT
+	bool
+
+config BT_CTLR_DF_ANT_SWITCHING_1US_SUPPORT
+	bool
+
+menuconfig BT_CTLR_DF
+	bool "LE Direction Finding [Experimental]"
+	depends on BT_CTLR_DF_SUPPORT
+	help
+	  Enable support for Bluetooth 5.1 Direction Finding
+
+if BT_CTLR_DF
+
+# Basic controller functionalities required for implementation of
+# Bluetooth v5.1 Direction Finding
+
+config BT_CTLR_DF_CTE_TX
+	bool "Enable transmission of Constant Tone Extension"
+	depends on BT_CTLR_DF_CTE_TX_SUPPORT
+	default y
+	help
+	  Enable support for transmission of Constant Tone Extension in
+	  controller.
+
+config BT_CTLR_DF_ANT_SWITCHING_1US
```

Comment:
I think it would be better to use SWITCH consistent, we have mix of switch and switching.

For example switching_patterns_len and switch_pattern_len

Reply:
changed to: `BT_CTLR_DF_ANT_SWITCH_TX`